src/main/trait.ts

```diff
@@ -12,7 +12,13 @@ export default function traits<T extends Array<Class<any>>> (constructors: T): C
   const cls = class {
     constructor () {
       constructors.forEach((c: any) => {
-        Reflect.construct(c, [], this)
+        const tmp = Reflect.construct(c, [], this)
+        for (let key in Object.keys(tmp)) {
```

Comment:
Oops, this for should be `for..of`, not `in`. I’ll fix it later, once I’m at home again.